apps/web/src/components/Layout/Nav/DesktopNav.tsx

```diff
@@ -19,6 +20,14 @@ type DropdownLinkProps = {
 };
 
 function DropdownLink({ href, label, color, externalLink }: DropdownLinkProps) {
+  const { logEventWithContext } = useAnalytics();
+
+  const handleClick = useCallback(() => {
+    logEventWithContext(label.replace(/[\/\\\- \n]/g, '_'), ActionType.click, {
```

Comment:
Nitpick: instead of doing string manipulation we could add an additional props to DropdownLink, say `eventeName` or something -- that way we change rename the even label without breaking the analytics relationship.